catalog/util.sh

```diff
@@ -25,7 +25,7 @@ function createPackage() {
     else
         CMD_ARRAY=("$OPENWHISK_HOME/bin/go-cli/wsk" -i --apihost "$EDGE_HOST" package update --auth "$AUTH_KEY" --shared yes "$PACKAGE_NAME" "${REST[@]}")
     fi
-    export WSK_CONFIG_FILE= # override local property file to avoid namespace clashes
+    export WSK_CONFIG_FILE=None # override local property file to avoid namespace clashes
```

Comment:
I don't see that we have a test for this (would require saving ~/.wskprops, creating a new one with a namespace set, running a wsk command with WSK_CONFIG_FILE override, and confirming that namespace from wskprops is not read, then restoring ~/.wskprops (unconditionally)).